src/keyword_vpd_parser.cpp

```diff
@@ -0,0 +1,143 @@
+#include "keyword_vpd_parser.hpp"
+
+#include "constants.hpp"
+#include "exceptions.hpp"
+
+#include <iostream>
+#include <numeric>
+#include <string>
+
+namespace vpd
+{
+
+/* @brief Keyword VPD parser function*/
```

Comment:
Remove this from here. We have it in hpp file.

Reply:
yes